### ISSUE TITLE
Like count doesn't show immediately after liking a note

### ISSUE THREAD

**Issue by ananyo2012** [publiclab/plots2#557]
### What happened just before the problem occurred
When a research note is liked with the like button  the like count doesn't show immediately.It shows only after the page is refreshed.

### Relevant URLs
https://publiclab.org/notes/warren/05-27-2016/recent-updates-on-publiclab-org-google-summer-of-code

### PublicLab.org username
ananyo2012

### Browser, version, and operating system
Mozilla firefox, Google Chrome

**Comment by jywarren** [comment]
Ah, to clarify, do you mean that the star gets filled but the number
doesn't increase?

Perhaps we need a JavaScript test suite for this kind of thing. What do you
think?

**Comment by ananyo2012** [comment]
@jywarren Yeah that would be great!

**Comment by jywarren** [comment]
Could you open a new issue for that, proposing Jasmine, perhaps?

**Comment by ananyo2012** [comment]
Yeah sure Jeff.Jasmine would be good. Suited for Ruby type projects.
